src/main/scala/firrtl/Emitter.scala

```diff
@@ -413,8 +413,10 @@ class VerilogEmitter extends SeqTransform with Emitter {
     val assigns = ArrayBuffer[Seq[Any]]()
     val attachSynAssigns = ArrayBuffer.empty[Seq[Any]]
     val attachAliases = ArrayBuffer.empty[Seq[Any]]
-    // Always blocks, keyed by (clock, asyncReset)
-    val alwaysBlocks = mutable.LinkedHashMap[(Expression, Option[Expression]), ArrayBuffer[Seq[Any]]]()
+    // No (aka synchronous) always blocks, keyed by clock
+    val noResetAlwaysBlocks = mutable.LinkedHashMap[Expression, ArrayBuffer[Seq[Any]]]()
+    // One always block per async reset register, (Clock, Reset, Content)
+    val asyncResetAlywaysBlocks = mutable.ArrayBuffer[(Expression, Expression, Seq[Any])]()
```

Comment:
Spelling, `alyways` -> `always`. Also, this probably needs some comment as to why you're doing it this way. I'd also accept a rant in place of a comment.

Reply:
Done